src/components/datepicker/DatePickerInput.tsx

```diff
@@ -0,0 +1,197 @@
+import React, { useEffect, useState } from 'react';
+import Box, { pickBoxProps } from '../box';
+import DatePicker from '.';
+import Icon from '../icon';
+import Input from '../input';
+import Popover from '../popover';
+import cx from 'classnames';
+import theme from './theme.css';
+import { formatDate } from './localeUtils';
+import { IconCalendarSmallOutline } from '@teamleader/ui-icons';
+import { DateTimeFormatOptions } from 'luxon';
+import { InputProps } from 'react-select';
```

Comment:
This should be our own Input props from `'../input'` (which doesn't exist yet, I guess). Maybe just use `TSFixMe` and add a todo.